<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>survival_mode</class>
 <widget class="QWidget" name="survival_mode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qradialgradient(spread:repeat, cx:0.5, cy:0.5, radius:0.077, fx:0.5, fy:0.5, stop:0 rgba(0, 0, 0, 255), stop:0.497326 rgba(0, 0, 0, 147), stop:1 rgba(0, 169, 255, 147));</string>
  </property>
  <widget class="QStackedWidget" name="stacked">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>511</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:1, stop:0 rgba(28, 95, 180, 255), stop:1 rgba(0, 0, 0, 255));
color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>381</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(191, 64, 64, 0);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:28pt; font-weight:600;&quot;&gt;Enter the difficulty&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="dif_e">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>200</y>
       <width>461</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10 px;
border-color: black;
font: 57 16pt &quot;Ubuntu Sans&quot;;
color: rgb(255, 255, 255);
background-color: rgb(224, 27, 36);</string>
     </property>
     <property name="text">
      <string>easy</string>
     </property>
    </widget>
    <widget class="QPushButton" name="dif_m">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>280</y>
       <width>461</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10 px;
background-color: rgb(192, 28, 40);
border-color: black;
font: 57 16pt &quot;Ubuntu Sans&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>medium</string>
     </property>
    </widget>
    <widget class="QPushButton" name="dif_h">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>360</y>
       <width>461</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10 px;
background-color: rgb(165, 29, 45);
border-color: black;
font: 57 16pt &quot;Ubuntu Sans&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>hard</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_5">
    <widget class="QLabel" name="turn">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>511</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(191, 64, 64, 0);
font: 81 20pt &quot;Ubuntu Sans&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;You only have 3 hearts&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="hint_1">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>220</y>
       <width>411</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(191, 64, 64, 0);
color: rgb(255, 255, 255);
font: 75 12pt &quot;Ubuntu Sans&quot;;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="turner_btn">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>390</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10px;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Let's go</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="ctr_1">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(192, 28, 40);</string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>781</width>
       <height>491</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:72pt; font-weight:600;&quot;&gt;3&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="ctr_2">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 194, 126);</string>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>791</width>
       <height>491</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:72pt; font-weight:600;&quot;&gt;2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="ctr_3">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(245, 194, 17);</string>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>791</width>
       <height>491</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:72pt; font-weight:600;&quot;&gt;1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_6">
    <widget class="QLabel" name="q_box">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>721</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);
border-radius: 10px;
color: rgb(153, 193, 241);
font: 57 12pt &quot;Ubuntu Sans&quot;;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QToolButton" name="q_ans1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>731</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="q_ans2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>270</y>
       <width>731</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:1, stop:0 rgba(28, 95, 180, 255), stop:1 rgba(0, 0, 0, 255));
border-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="q_ans3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>330</y>
       <width>731</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:1, stop:0 rgba(28, 95, 180, 255), stop:1 rgba(0, 0, 0, 255));
border-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="q_ans4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>390</y>
       <width>731</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:1, stop:0 rgba(28, 95, 180, 255), stop:1 rgba(0, 0, 0, 255));
border-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLabel" name="m_score">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(191, 64, 64, 0);
color: rgb(255, 255, 255);
font: 81 20pt &quot;Ubuntu Sans&quot;;</string>
     </property>
     <property name="text">
      <string>Score: </string>
     </property>
    </widget>
    <widget class="QProgressBar" name="timesh">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>20</y>
       <width>118</width>
       <height>23</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="maximum">
      <number>15</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="format">
      <string>%v</string>
     </property>
    </widget>
    <widget class="QLabel" name="heart1">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>10</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(191, 64, 64, 0);
font: 20pt &quot;Ubuntu Sans&quot;;
color: rgb(224, 27, 36);</string>
     </property>
     <property name="text">
      <string>❤️</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="heart2">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>10</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(191, 64, 64, 0);
font: 20pt &quot;Ubuntu Sans&quot;;
color: rgb(224, 27, 36);</string>
     </property>
     <property name="text">
      <string>❤️</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="heart3">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>10</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(191, 64, 64, 0);
font: 20pt &quot;Ubuntu Sans&quot;;
color: rgb(224, 27, 36);</string>
     </property>
     <property name="text">
      <string>❤️</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="t_ans">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(51, 209, 122);</string>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 81 20pt &quot;Ubuntu Sans&quot;;</string>
     </property>
     <property name="text">
      <string>Correct answer</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="hint_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>210</y>
       <width>491</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QToolButton" name="t_wid">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>390</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 44, 142);
border-radius:10px;</string>
     </property>
     <property name="text">
      <string>continue</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="f_ans">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(224, 27, 36);</string>
    </property>
    <widget class="QLabel" name="f_disp">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>40</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 81 20pt &quot;Ubuntu Sans&quot;;</string>
     </property>
     <property name="text">
      <string>Wrong answer</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="hint_3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>220</y>
       <width>491</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QToolButton" name="f_wid">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>390</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 44, 142);
border-radius:10px;</string>
     </property>
     <property name="text">
      <string>continue</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(192, 28, 40);</string>
    </property>
    <widget class="QLabel" name="go_l1">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>50</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:26pt; font-weight:600;&quot;&gt;Game&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="go_l2">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>50</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:26pt; font-weight:600;&quot;&gt;over&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="go_score">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>200</y>
       <width>341</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;Ubuntu Sans&quot;;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="go_dif">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>330</y>
       <width>341</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Ubuntu Sans&quot;;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
